safenode/src/domain/storage/spends.rs

```diff
@@ -101,20 +101,22 @@ impl SpendStorage {
         if let Ok(existing) = self.get(&address).await {
             let tamper_attempted = signed_spend.spend.hash() != existing.spend.hash();
             if tamper_attempted {
-                // We don't error if the double spend failed, as we rather want to
-                // announce the double spend attempt to close group. TODO: how to handle the error then?
-                self.try_store_double_spend(&existing, signed_spend).await?;
-
-                // The spend is now permanently removed from the valid spends.
-                // We don't error if the remove failed, as we rather want to
-                // announce the double spend attempt to close group.
-                // The double spend will still be detected by querying for the spend.
-                self.remove(&address, &self.valid_spends_path).await?;
-
-                return Err(Error::DoubleSpendAttempt {
-                    new: Box::new(signed_spend.clone()),
-                    existing: Box::new(existing),
-                });
+                trace!("Tamper attempt detected, jsut rejecting the request.");
+                return Err(Error::AlreadyExists(address));
```

Comment:
no longer `DoubleSpendAttempt` error to be returned, 
which will trigger a further handling within the caller?
https://github.com/maidsafe/safe_network/pull/197/files#diff-ae93be614203f23a4b33e38e2ac1aafcf961ae7d4e6ac9485fae33eb475e0938R326

Reply:
Yes, it is disabled for now.

If it exists it exists. The client can tryt double spend but it will always be rejected, because we have the state of the first spend.

The double spend state is a way to _punish_ the double spending client, not to _hinder_ it. It is already hindered.